### DISCUSSION TITLE
Can I get a definitive example of how to configure esLint for nuxt3?

### DISCUSSION THREAD

**Discussion by suityou01** [nuxt/nuxt#28590] [Questions]
The example given in the documentation is really poor. I have tried everything I can think of to set it up correctly.

First I did this

```bash
npx nuxi module add eslint
```

Then I added some ignores to my config, in  eslint.config.js which is just ignored.

In my project I have 2 top level folders I do not with the linter to traverse. /docker and /tests

So in my config I have 

```
ignores: ["**/*.config.js", "!**/eslint.config.js", "docker/", "tests/"],
```

Which according to the EsLint documentation should work right out of the box for me.

I would imagine others would be in the same position as not every folder contains things you want to be linted. .nuxt for example.

Can we get a definitive example of how to set up linting with Nuxt.js?

**Comment by AloisSeckar** [comment]
I have a demo repo with working Nuxt+ESlint integration here - https://github.com/AloisSeckar/demos-nuxt/tree/main/nuxt-simple

It works pretty simple via the [Nuxt ESLint module](https://eslint.nuxt.com/packages/module). Just for some reason the last fully working version was `0.3.13`. I keep having issues with v0.4.0 and v0.5.0 released recently.

Your `ignores` option would go into config object of `withNuxt` method of  `eslint.config.mjs` file.
